src/6model/reprs/Decoder.c

```diff
@@ -124,6 +124,29 @@ static int should_translate_newlines(MVMThreadContext *tc, MVMObject *config) {
     }
     return 0;
 }
+static MVMString * has_replacement(MVMThreadContext *tc, MVMObject *config) {
+    char *replacement_const = "replacement";
+    int replacement_const_length = strlen(replacement_const);
+    if (IS_CONCRETE(config) && REPR(config)->ID == MVM_REPR_ID_MVMHash) {
+        MVMObject *value = MVM_repr_at_key_o(tc, config,
+            MVM_string_ascii_decode(tc, tc->instance->VMString, replacement_const, replacement_const_length));
```

Comment:
Please add this to the `str_consts` in instance. Otherwise `MVM_string_ascii_decode` may trigger GC. We can `MVMROOT` instead, but faster to not do this work again and again.

Reply:
Yes that is something I was going to do before finalizing this.